runtime/bajun/Cargo.toml

```diff
@@ -148,6 +152,7 @@ runtime-benchmarks = [
     "xcm-builder/runtime-benchmarks",
     "cumulus-pallet-session-benchmarking/runtime-benchmarks",
     "cumulus-pallet-xcmp-queue/runtime-benchmarks",
+	"pallet-ajuna-awesome-avatars/runtime-benchmarks",
```

Comment:
We need to enable `pallet-ajuna-awesome-avatars/std` feature as well

Reply:
Done